packages/e2e-testing/__tests__/bad-network.test.ts

```diff
@@ -97,18 +104,41 @@ afterAll(async () => {
 
 describe('Payment & Receipt Managers E2E', () => {
   test.only(`Can create channels and make 20 payments`, async () => {
-    const allocations = generateAllocationIdAndKeys(5);
+    const allocations = generateAllocations(NUM_ALLOCATIONS);
+
+    await syncAllocations(PAYER_SERVER_URL, {
+      requests: allocations.map((allocation) => ({
+        allocation: {
+          id: allocation.id,
+          indexer: {
+            id: allocation.indexer.id,
+            url: allocation.indexer.url,
+            stakedTokens: allocation.indexer.stakedTokens
+          },
+          subgraphDeploymentID: {
+            ...allocation.subgraphDeploymentID,
+            display: allocation.subgraphDeploymentID.display,
+            ipfsHash: allocation.subgraphDeploymentID.ipfsHash,
+            bytes32: allocation.subgraphDeploymentID.bytes32
+          },
+          allocatedTokens: allocation.allocatedTokens,
+          createdAtEpoch: allocation.createdAtEpoch
+        },
+        num: NUM_ALLOCATIONS,
+        type: 'SetTo'
```

Comment:
I thought there was a fixture for this? Let me check.

Reply:
A (local) fixture is used [here](https://github.com/statechannels/graph-payments/blob/247b22941132b92fe676e594ef40328abea2ab7b/packages/payments/src/__tests__/extract-capacities.test.ts#L11).

Reply:
Since it is not that useful and in a different package, I'd like to opt to not re-use this for now until there is a 3rd use case warranting a generalization.